modules/rustbitcoin/rust_bitcoin_lib/src/lib.rs

```diff
@@ -6,7 +6,11 @@ use std::str::{FromStr, Utf8Error};
 
 use bitcoin::address::Address;
 use bitcoin::consensus::{deserialize_partial, encode, serialize};
+use bitcoin::consensus::Decodable;
+use bitcoin::consensus::{deserialize_partial, encode};
```

Comment:
Why are you importing `deserialize_partial` and `encode` again? I think this is causing CI to fail.